README.md

```diff
@@ -27,6 +27,109 @@ yarn okapi
 vagrant up release
 ```
 
+## Building images for deployment
+
+Update `Vagrantfile` with a synced directory to fw-cli for mod-workflow and mod-camunda development.
+
+```
+snapshot.vm.synced_folder "C:/Users/FOLIO/Development/work/FOLIO/fw-cli", "/home/vagrant/fw-cli", owner: "vagrant", group: "vagrant", mount_options: ["uid=1000", "gid=1000"]
```

Comment:
Could this path be renamed to prevent exposure of `C:/Users/FOLIO/Development/work/FOLIO/fw-cli`
a relative path could be provided.